test/PayID/pay-id-client-test.ts

```diff
@@ -176,4 +182,72 @@ describe('Pay ID Client', function(): void {
       done()
     })
   })
+
+  // getInvoice
+
+  it('getInvoice - successful response', async function() {
+    // GIVEN a PayID client, valid PayID and mocked networking to return a invoice for the Pay ID.
+    const payID = '$xpring.money/georgewashington'
+    const payIDClient = new PayIDClient(XRPLNetwork.Test)
+
+    const mockResponse = {
+      messageType: 'Invoice',
+      message: {
+        nonce,
+        expirationTime: '2020-03-18T04:04:02',
```

Comment:
number now

Reply:
Talked offline, leaving this as a string. Updated to unix timestamp format.